36/upgrade.html

```diff
@@ -116,6 +116,12 @@ <h5><a id="upgrade_360_notable" href="#upgrade_360_notable">Notable changes in 3
             For more information about the early access tiered storage feature, please check <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-405%3A+Kafka+Tiered+Storage">KIP-405</a> and
             <a href="https://cwiki.apache.org/confluence/display/KAFKA/Kafka+Tiered+Storage+Early+Access+Release+Notes">Tiered Storage Early Access Release Note</a>.
         </li>
+        <li>Transaction partition verification (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-890%3A+Transactions+Server-Side+Defense">KIP-890</a>)
+            has been added to data partitions to prevent hanging transactions. Workloads with compression can experience InvalidRecordExceptions and UnknownServerExceptions.
+            For workloads with compression, <code>transaction.partition.verification.enable</code> should be set to false. Note that the default for 3.6 is true.
```

Comment:
I'd just say "This feature can be disabled by setting..." (instead of "For workloads with compression...".